Makefile

```diff
@@ -59,6 +63,6 @@ test:
 .NOTPARALLEL: test
 .PHONY: test
 
-edoc:
-	@rebar3 edoc
-.PHONY: edoc
+ex_doc-dry:
+	@rebar3 hex publish docs --dry-run
+.PHONY: ex_doc-dry
```

Comment:
^ I'm assuming there's no need for an `ex_doc` target because `rebar3_hex` takes care of doc generation on the new `rebar.config`